devel/mos/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+categories          devel
+license             Apache-2.0
+name                mos
+
+set python.versions \
+                    38
+
+depends_lib-append  port:libftdi1 \
+                    port:libftdi0 \
+                    port:libusb \
+                    port:libusb-compat \
+
+depends_build-append \
+                    port:python${python.versions} \
+                    port:py${python.versions}-setuptools \
+                    port:pkgconfig \
+
+maintainers         {@harens gmail.com:harensdeveloper} \
+                    openmaintainer
+description         Mongoose OS command-line tool
+long_description    ${description}. Provides device management \
+                    and firmware building capabilities.
+
+subport ${name}-latest {
+    long_description \
+                    A port of ${name} aimed for development purposes
+}
+
+if {${subport} eq "${name}"} {
+    conflicts       ${name}-latest
+    go.setup        github.com/mongoose-os/mos 2.18.0
+    checksums       rmd160  9ad2092f04bae9e21d26999e38699c5d3ceba7e3 \
+                    sha256  555df9fc6a629f6b79e61245cb4802544193e14528b110c13de015e2098954be \
+                    size    673863
+} else {
+    conflicts       ${name}
+    go.setup        github.com/mongoose-os/mos 93ecd2a813fd936d521f3f5b15f09f6140d850dc
```

Comment:
This subport has not defined a version number. If this commit does not correspond to a specific version number, you could append a YYYYMMDD string (for the date of the commit) to the version number, such as `version 1.2.3-20200915`